spec/features/job_seekers_can_view_a_vacancy_spec.rb

```diff
@@ -91,6 +97,90 @@
       expect(page).to have_content(30)
     end
 
+    context 'when the upload documents feature flag is OFF' do
+      before do
+        vacancy = create(:vacancy, :published)
+        visit job_path(vacancy)
```

Comment:
This `before` block, and the 3 other new `before` blocks, are probably time-expensive when running the test suite, as they run before each example. You might want to refactor each of these 4 `context`s to only have one `scenario` instead of 2, so that each `context` only has to run the `before` stuff once. I recognise that reduces the tidiness of the code, though.